tests/test_managers_mail_manager.py

```diff
@@ -1,4 +1,4 @@
-from config import mail
+from config import mail as mail_config
```

Comment:
having to do this sort of draws a red flag about the naming conflicts

Reply:
Ahh good point! I will get this changed!